inst/get-testdata/score_untrained-mean.R

```diff
@@ -0,0 +1,187 @@
+## Data for testing the score_untrained example: mean output
+# load the package to make its internal functions available
+devtools::load_all()
+## target data
+target_data_mean <- readRDS(
+  testthat::test_path("testdata/flu_example_target_data.rds")
+) |> mutate(target_end_date = as.Date(target_end_date))
+
+## forecast data with mean output
+forecast_mean <- readRDS(
+  testthat::test_path("testdata/flu_example_mean_model_output.rds")
+)
+valid_tbl_mean <- validate_input_data(forecast_mean, target_data_mean)
+mean_data_list <- split_data_by_task(valid_tbl_mean,
+  weighted = FALSE,
+  training_window_length = 0
+)
+
+# replace with simple values for easy calculation
+dat_mean1 <- mean_data_list[[16]]
+dat_mean <- rbind(dat_mean1, dat_mean1[1:2, ])
+dat_mean$model_id <- c(dat_mean1$model_id, "fake-mod1", "fake-mod2")
+dat_mean$value <- c(30, 12, 18, 6, 6)
+
+model_id_list <- unique(dat_mean$model_id)
+# find index of target_data corresponding to dat
+idx <- with(
+  target_data_mean,
+  target_end_date == unique(dat_mean$target_end_date) &
+    output_type == unique(dat_mean$output_type) &
+    location == unique(dat_mean$location)
+)
+# replace the target oracle value with a simple value
+target_data_mean$oracle_value[idx] <- 10
+
+## Case 1: no missing data and 'linear pool' ensemble
+# ensemble built from all models
+ens_df <- linear_pool(dat_mean, model_id = "ens_all")
+# construct ensembles without each model
+model_names <- dat_mean$model_id
+for (i in seq_along(model_names)) {
+  sub_data <- dat_mean[model_names != model_names[i], ]
+  sub_ens <- linear_pool(sub_data,
+    model_id = paste0("ens_wo_", model_names[i])
+  )
+  ens_df <- bind_rows(ens_df, sub_ens)
+}
+# evaluate each ensemble
+ens_df <- score_model_out(ens_df,
+  target_data_mean,
+  metrics = "se_point"
+) |>
+  # calculate importance scores: subtract the error of the ensemble-all
+  mutate(importance = se_point - se_point[1]) |>
+  filter(model_id != "ens_all")
+ens_df$model_id <- sub("^ens_wo_", "", ens_df$model_id)
```

Comment:
I walked through this code and I think the logic is sound. One small comment about making this code more readable would be to not use and then reused simple not explanatory names like `ens_df`. This is used twice in the above chunk of code (and similarly in the other cases below) for different things. First it is used to store an ensemble, then it is used to store the scores. I think it would be better practice to use longer and more clear names so that someone new to the code could better understand what each piece is doing.